<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeWindow</class>
 <widget class="QMainWindow" name="RealTimeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>207</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Real-Time Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nexus.qrc">
    <normaloff>:/images/favicon.png</normaloff>:/images/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	outline:none;
	gridline-color: #252933;
	background: #15181f;
	font:11px Arial;
}
QMainWindow{
	background: #15181f;
}
#centralwidget {
	background: #15181f;
}
QHeaderView::section{
	font-size: 11px;
	color:#fff;
	padding:5px;
	border:none;
	border-right:1px solid #252933;
	border-bottom:1px solid #252933;
	background: #15181f;
	font-weight:bold;
}
QTableCornerButton::section{
	border:none;
	background: #15181f;
}
QTableView::item{
	color:#ccd3e5;
	padding:2px;
	background-color:#15181f
}


QScrollBar {
    border: none;
    background: #20252e;
	border-radius:2px;
    width: 5px;
    margin: 0;
}

QScrollBar::handle {
    background: rgba(255,255,255,0.1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::handle:hover {
    background: rgba(36,219,161,1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::sub-line, QScrollBar::add-line{
    subcontrol-origin: margin;
    height: 20px;
    border: none;
    background: transparent;
	border-radius:2px;
}

QScrollBar::up-arrow, QScrollBar::down-arrowl {
    border: none;
}

QScrollBar::add-page, QScrollBar::sub-page {
    background: none;
}
QPushButton#configBtn {
	min-width:22px;
	height:14px;
}
QPushButton {
    background: #2574ab;
    border-radius: 2px;
	padding: 3px 5px 2px;
    border: none;
    min-width: 100px;
    color: #ffffff;
}

QPushButton::hover {
    background: #2b85c4;
}

QPushButton::pressed {
    background: #206291;
}

QPushButton:disabled {
    background-color: #1c2029;
    color: #46474a;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="playerTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:#15181f</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>8</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Handle</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DPS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dmg Out</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dmg In</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Shots</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Misses</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Crits</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kills</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Deaths</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Accuracy</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="11">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="uploadBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Upload</string>
          </property>
          <property name="icon">
           <iconset resource="../nexus.qrc">
            <normaloff>:/images/upload.png</normaloff>
            <disabledoff>:/images/upload_disabled.png</disabledoff>:/images/upload.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset resource="../nexus.qrc">
            <normaloff>:/images/reset.png</normaloff>
            <disabledoff>:/images/reset_disabled.png</disabledoff>:/images/reset.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configBtn">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../nexus.qrc">
            <normaloff>:/images/monitor-config.png</normaloff>:/images/monitor-config.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../nexus.qrc"/>
 </resources>
 <connections/>
</ui>
